src/eventSchemas/models/schemaCodeGenUtils.ts

```diff
@@ -0,0 +1,93 @@
+/*!
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// TODO: This is fragile. Very fragile. But it is necessary to get Schemas service launched, and we've evaluated all other tradeoffs
```

Comment:
Just a matter of preference. It is certainly a small class, but it does have a dedicated benefit. It encapsulates the knowledge of building schema package name for  AWS, partner and custom events. It is easy to mock&test,  also matches with the implementation used in Intellij. It is also worth mentioning that these hyper-focused classes will be replaced by api calls.

Reply:
Applies to the two classes below as well.

Reply:
When we are downloading schemas code for a specific event, we are also importing strongly typed event interface from it into the lambdaHandlers(app.py) created by sam cli. If this class doesn't properly build the parameters, then incorrect class names could be imported into the lambdaHandlers instead of the actual generated&downloaded schemas classes. It is fragile because we are building these parameters from different chain of json keys value for each event type + doing filtering, replacing characters before finalizing the value.  That being said, it is also worth mentioning that this already got approved by Intellij  toolkit team, including Kyle.